atlassian_jwt_auth/contrib/flask_app/utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+import jwt.exceptions
+import requests.exceptions
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
```

Comment:
Yea, I know that I submitted this code... but this is a direct copy & paste from the `asap.py` file in Instant Micros :cry: 

Not saying that I'm opposed to changing it, but just establishing why this is the way it is